planet/reporting.py

```diff
@@ -110,3 +110,48 @@ def update(self,
 
         if self.bar is not None:
             self.bar.refresh()
+
+
+class AssetStatusBar(ProgressBar):
+    """Bar reporter of asset status."""
+
+    def __init__(
+        self,
+        item_type,
+        item_id,
+        asset_type,
```

Comment:
Might be worth doing this:
```suggestion
        item_type: str,
        item_id: str,
        asset_type: str,
```